packages/fx-core/src/plugins/solution/fx-solution/debug/scaffolding.ts

```diff
@@ -238,15 +244,15 @@ export async function _scaffoldLocalDebugSettings(
     TelemetryEventName.scaffoldLocalDebugSettings,
     telemetryProperties
   );
-  return ok(Void);
+  return ok(localSettings as Json);
```

Comment:
The declared return type of this function is `Promise<Result<Void, FxError>>`, why return `ok(localSettings as Json)` here?